Rubberduck.Parsing/ComReflection/ComProject.cs

```diff
@@ -146,6 +146,18 @@ private void LoadModules(ITypeLib typeLibrary)
                 }
                 catch (COMException) { }
             }
+            ApplySpecificLibraryTweaks();
+        }
+
+        private void ApplySpecificLibraryTweaks()
+        {
+            if (!Name.ToUpper().Equals("EXCEL")) return;
+            var application = _classes.SingleOrDefault(x => x.Guid.ToString().Equals("00024500-0000-0000-c000-000000000046"));
+            var worksheetFunction = _interfaces.SingleOrDefault(i => i.Guid.ToString().Equals("00020845-0000-0000-c000-000000000046"));
```

Comment:
Yeah, ultimately - maybe something like a `Dictionary<ComCoClass, ComInterface> InterfaceAdditions`. I'm sure there's some Word and Access stuff we don't know about yet.